an-overview-of-the-system-database.adoc

```diff
@@ -0,0 +1,62 @@
+= An overview of the system database
+:slug: an-overview-of-the-system-database
+:author: Vivek Saran
+:twitter: @your-twitter
+:neo4j-versions: 4.0, 4.1
+:tags: system
+:category: operations
+
+Neo4j 4.0 and higher versions support the management of multiple databases within the same DBMS. All these databases are controlled through a special database called the `system` database.
+
+This article provides a brief architectural overview of the `system` database.
+
+The role of the `system` database is to **define the configuration** for the other databases. There could be various types of configuration for the databases. For example:
+
+*	operational configuration
+    -	existence (exists or not)
+    -	status (online/offline)
+*	security configuration (RBAC)
+* neo4j,conf  (this is not yet maintained by the `system` database, but there are plans in the future versions)
```

Comment:
The canonical syntax in AsciiDoc would be:

```suggestion
* operational configuration
** existence (exists or not)
** status (online/offline)
* security configuration (RBAC)
* neo4j,conf (this is not yet maintained by the `system` database, but there are plans in the future versions)
```

But what you wrote also works :smile: